src/Ionide.ProjInfo/Library.fs

```diff
@@ -1212,6 +1304,32 @@ type WorkspaceLoader private (toolsPath: ToolsPath, ?globalProperties: (string *
                 | Error msg ->
                     loadingNotification.Trigger(WorkspaceProjectState.Failed(p, GenericError(p, msg)))
                     [], None
+                | Ok project ->
+                    let mappedProjectInfo = ProjectLoader.getLoadedProjectInfo p customProperties project
+
+                    match mappedProjectInfo with
+                    | Ok project ->
+                        try
+                            cache.Add(p, project)
+
+                            let lst =
+                                project.ReferencedProjects
+                                |> Seq.choose (fun n ->
+                                    if cache.ContainsKey n.ProjectFileName then
+                                        None
+                                    else
+                                        Some n.ProjectFileName
+                                )
+                                |> Seq.toList
+
+                            let info = Some project
+                            lst, info
+                        with exc ->
```

Comment:
Worth logging exception so we get a stacktrace.

Reply:
there's no longer any throwing on this code path, so I was able to remove this try/catch entirely.